.github/workflows/pull-request-image.yml

```diff
@@ -0,0 +1,107 @@
+name: Create Docker image for local testing
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Build and archive plugin build artifacts
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup packages
+        id: setup-packages
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y zip zstd jq git
```

Comment:
`git`, `zip`, and `jq` should be preinstalled on the Ubuntu image (see [here](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md))